packages/cli/templates/vue3/src/components/FieldPluginExample/index.vue

```diff
@@ -41,7 +37,7 @@ function closeModal() {
     <div class="container">
       <Counter
         :count="plugin.data.content"
-        :onIncrease="() => plugin.actions.setContent(plugin.data.content + 1)"
+        @onIncrease="plugin.actions.setContent(plugin.data.content + 1)"
```

Comment:
I needed to make some changes to fix typescript issues. I will attach a video to explain why there was a TS issue in vue3 template.